themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -68,9 +69,10 @@
 
         {block name='form_field_item_radio'}
           {foreach from=$field.availableValues item="label" key="value"}
-            <label class="radio-inline">
+            <label class="radio-inline" for="{$field.name}">
```

Comment:
Do we need to add the `for` even if the `input` is inside the `<label>` ?

Reply:
I don't think so